modules/installation-aws-config-yaml.adoc

```diff
@@ -367,6 +366,9 @@ endif::vpc,restricted[]
 ====
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
 ====
+ifndef::vpc,restricted[]
+<12> You can provide either the `NLB` or `Classic` load balancer type in this parameter. If you do not specify a value, the installation program defaults to `Classic`.
```

Comment:
Is there a particular reason you're adding this as callout number 12?

Usually we'd want the callouts to go in order, so where this is now it should be number 6. Unless there's a reason it needs to be like this due to the crazy ifdefs?

Reply:
Yes, it's the crazy ifdefs. I couldn't change them all.

Reply:
It's probably something we should fix at some point, but honestly I'd rather it be out of order than throw things off due to how complex this is. Let's merge and put it on the back burner to look at another day!